src/module/data-localstorage.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable class-methods-use-this */
+export default class localStorageList {
+  git(key) {
+    return localStorage.getItem(key);
+  }
+
+  sit(key, val) {
+    localStorage.setItem(key, val);
+  }
+}
```

Comment:
Great job so far 
I suggest deleting this class because you already use one doing the same job so delete it, please.